centreon/src/Core/Dashboard/Playlist/Domain/Model/PlaylistContactGroupShare.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace Core\Dashboard\Playlist\Domain\Model;
+
+use Centreon\Domain\Common\Assertion\Assertion;
+
+class PlaylistContactGroupShare
+{
+    public const PLAYLIST_VIEWER_ROLE = 'viewer';
+    public const PLAYLIST_EDITOR_ROLE = 'editor';
+    public const PLAYLIST_ROLES = [self::PLAYLIST_VIEWER_ROLE, self::PLAYLIST_EDITOR_ROLE];
```

Comment:
Could it be that this constants could be defined at PlaylistShare level ? This would avoid code duplication and might be clearer ? A playlist share as multiple shares level viewer, editor and so on... ?

An other idea also could be to use an Enum for this something like `PlaylistShareRoles` 👍🏻